src/Workspaces/Core/Portable/Workspace/Solution/Solution.cs

```diff
@@ -47,25 +47,23 @@ private Solution(SolutionCompilationState compilationState)
         {
         }
 
-        // TODO(cyrusn): Remove and update references to access this.State.
-        private SolutionState _state => this.State;
-        internal SolutionState State => CompilationState.Solution;
+        internal SolutionState SolutionState => CompilationState.Solution;
```

Comment:
Were you still planning on renaming CompilationState.Solution to CompilationState.SolutionState in a different PR?

Reply:
i'll do that now :)